src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/recordings/test_mysql_proxy_resources_mgmt.yaml

```diff
@@ -4002,7 +4002,7 @@ interactions:
         and can be used to control the operation of AUTO_INCREMENT columns.","defaultValue":"1","dataType":"Integer","allowedValues":"1-65535","source":"system-default","isConfigPendingRestart":"False","isDynamicConfig":"True"},"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.DBforMySQL/servers/azuredbclitest000002/configurations/auto_increment_increment","name":"auto_increment_increment","type":"Microsoft.DBforMySQL/servers/configurations"},{"properties":{"value":"1","description":"The
         auto_increment_offset is intended for use with source-to-source replication,
         and can be used to control the operation of AUTO_INCREMENT columns.","defaultValue":"1","dataType":"Integer","allowedValues":"1-65535","source":"system-default","isConfigPendingRestart":"False","isDynamicConfig":"True"},"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.DBforMySQL/servers/azuredbclitest000002/configurations/auto_increment_offset","name":"auto_increment_offset","type":"Microsoft.DBforMySQL/servers/configurations"},{"properties":{"value":"0","description":"Number
-        of seconds to service waits before the binary log file gets purged. See: https://docs.microsoft.com/en-us/azure/mysql/flexible-server/concepts-server-parameters#binlog_expire_logs_seconds","defaultValue":"0","dataType":"Integer","allowedValues":"0-4294967295","source":"system-default","isConfigPendingRestart":"False","isDynamicConfig":"True"},"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.DBforMySQL/servers/azuredbclitest000002/configurations/binlog_expire_logs_seconds","name":"binlog_expire_logs_seconds","type":"Microsoft.DBforMySQL/servers/configurations"},{"properties":{"value":"0","description":"Controls
+        of seconds to service waits before the binary log file gets purged. See: https://learn.microsoft.com/en-us/azure/mysql/flexible-server/concepts-server-parameters#binlog_expire_logs_seconds","defaultValue":"0","dataType":"Integer","allowedValues":"0-4294967295","source":"system-default","isConfigPendingRestart":"False","isDynamicConfig":"True"},"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.DBforMySQL/servers/azuredbclitest000002/configurations/binlog_expire_logs_seconds","name":"binlog_expire_logs_seconds","type":"Microsoft.DBforMySQL/servers/configurations"},{"properties":{"value":"0","description":"Controls
```

Comment:
Recordings should be skipped. As previously motioned in this PR: https://github.com/Azure/azure-cli/pull/27040.